sys-power/ncpufreqd/ncpufreqd-2.4.ebuild

```diff
@@ -7,8 +7,8 @@ EAPI="5"
 inherit cmake-utils
 
 DESCRIPTION="Daemon controlling CPU speed and temperature"
-HOMEPAGE="http://projects.simpledesigns.com.pl/project/ncpufreqd/"
-SRC_URI="http://projects.simpledesigns.com.pl/get/${P}.tar.bz2"
+HOMEPAGE="https://bitbucket.org/nelchael/ncpufreqd"
+SRC_URI="https://bitbucket.org/nelchael/${PN}/get/${P}.tar.bz2"
```

Comment:
Now here's the tricky part: we can't really use tag-generated tarballs like this. They may be generated on the fly, and so you can't trust having the same checksum on every download.

Now, if you were a developer I'd just ask you to mirror the file in your devspace. However, you can just ask Krzysiek to provide a downloadable tarball on bitbucket or elsewhere. He used to be a Gentoo developer so he should understand the issue :).